jdkhttp/src/main/java/org/kie/hacep/endpoint/bootstrap/JdkHttpServer.java

```diff
@@ -64,7 +64,7 @@ public void handle(HttpExchange httpExchange) throws IOException {
     }
 
     private static void initResponse(HttpExchange httpExchange, boolean isOk) throws IOException {
-        int returnCode = isOk ? 200 : 400;
+        int returnCode = isOk ? 200 : 503;
```

Comment:
Why have you changed from 400 to 503? Does this matter on k8s side?

Reply:
Seems correct service unavailable instead of bad request  btw : "Any code greater than or equal to 200 and less than 400 indicates success. Any other code indicates failure." https://v1-14.docs.kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/